InventoryWatch/Countries.swift

```diff
@@ -51,6 +51,19 @@ struct Country: Hashable {
             default:
                 return nil
             }
+        case .AirPodsProGen2:
+            switch self.name {
+            case "United States", "Canada":
+                return "AM"
+            case "Germany", "United Kingdom", "France", "Austria", "Netherlands":
+                return "ZM"
+            case "Australia", "Thailand":
+                return "ZA"
+            case "Italy":
+                return "TY"
+            default:
+                return nil
+            }
```

Comment:
not sure why AirPods Pro (2nd gen) SKUs are so different to previous schemes...?